dot/core/helpers_test.go

```diff
@@ -172,7 +172,7 @@ func NewTestService(t *testing.T, cfg *Config) *Service {
 	var stateSrvc *state.Service
 	testDatadirPath := t.TempDir()
 
-	gen, genesisTrie, genesisHeader := newTestGenesisWithTrieAndHeader(t)
+	gen, genesisTrie, genesisHeader := newWestendLocalWithTrieAndHeader(t)
```

Comment:
There wont be any issues with using local instead of dev here?

Reply:
No, the only difference between `westend-dev` and `westend-local` is the number of authorities in the json spec file but all use the same `westend` so for the `dot/core` package tests while the runtime keeps the same we can use `westend-dev|runtime` without problems

Reply:
awesome! Just wanted to double check